File: Python/Day1/simple_arithmetic_app.py
from random import *
from functions import *

score = 0
level = 1

while( level <= 10 ):
	first_number = randrange(1, 2000) + 0
	second_number = randrange(1, 500) + 0
	if second_number > first_number:
			temp = first_number
			temp_2 = second_number
			
			second_number = temp
			first_number = temp_2
			



	correct_answer = first_number - second_number

	answer = int(input(f"What is {first_number} - {second_number}?"))
	if correct_answer == answer:
		score += 1
	else:
		answer = int(input(print("Wrong answer... You have 1 trial left for this question... Try again: ")))
	if correct_answer == answer:
		print("correct!")
		score += 1
		
	else:
		print("Wrong Answer")
		
				
			

	level += 1
print(score)
	

File: Java/Day1/SimpleArithmeticApp.java
import java.util.Scanner;

public class SimpleArithmeticApp{
	public static void main(String[] args){
	System.out.print("hi");
	int score = 0;
	Scanner input = new Scanner(System.in);

	for( int level = 1; level <= 3; level++){
	int first_number = (int)(Math.random()*1000);
	int second_number = (int)(Math.random()*10);

	
	if( second_number > first_number){
		int temp = first_number;
		int temp_2 = second_number;

		second_number = temp;
		first_number = temp_2;
			}
	
	int correct_answer = first_number - second_number;

	System.out.printf("What is %d - %s? ", first_number, second_number);
	int answer = input.nextInt();
	if (correct_answer == answer)
		score++;
			
		else{
			System.out.print("Wrong answer... You have 1 trial left for this question... Try again: ");
			answer = input.nextInt();}

			if (correct_answer == answer){
				System.out.println("Correct!");
				score++;}
			else{
				System.out.println("Wrong answer");
				System.out.print("Would you like to continue?(y/n) ");}

		
}
System.out.printf("You scored %d / 10", score);}}
